cfme/infrastructure/virtual_machines.py

```diff
@@ -26,7 +26,9 @@
 from widgetastic_patternfly import Input as WInput
 
 from cfme.common import BaseLoggedInPage
+from cfme.common import ComparableMixin
 from cfme.common import TimelinesView
+from cfme.common.provider_views import TemplatesCompareView
```

Comment:
Shouldn't `TemplatesCompareView` also inherit from your base `CompareView`? Or do you want to make those changes in a separate PR?

Reply:
Done.